src/hotspot/share/runtime/arguments.cpp

```diff
@@ -2405,6 +2405,29 @@ jint Arguments::parse_each_vm_init_arg(const JavaVMInitArgs* args, bool* patch_m
         }
       }
 #endif // !INCLUDE_JVMTI
+    // SapMachine 2025-08-11: Handle jmc agent if requested.
+    } else if (match_option(option, "-jmcagent:", &tail)) {
+#if !defined(WITH_SAP_JMC_AGENT)
+      jio_fprintf(defaultStream::error_stream(),
+          "SAP JMC agent is not included in this VM\n");
+      return JNI_ERR;
+#else
+#if !INCLUDE_JVMTI
+#error "Must have JVMTI enabled with SAP JVM agent"
```

Comment:
Maybe stupid question, but is 'SAP JVM agent' really the correct naming (might apply also to the original change of course)?